lib/tasks/create_fake_data.rake

```diff
@@ -0,0 +1,89 @@
+namespace :db do
+  namespace :seed do
+    desc 'Generate fake patient entries for data wranglers'
+    task :create_fake_data => :environment do
+
+        users = User.all
+        clinics = Clinic.all
+
+        5000.times do |idx|
+          flag = (rand(10) % 5 == 0)
+
+          initial_call = Date.today - rand(1000)
+          
+          has_appt = rand(3) % 2 == 0
```

Comment:
Same deal for this and `has_pledge` - might behoove us to use the modulo of `idx` to be consistent. (I'm less worried about `initial_call` since there are so many possible values, but you could do it there too if you were interested in getting weird with it!)